.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,16 +17,27 @@ Reviewing guidelines: https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/do
 
 ###### Things done
 
-<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->
+<!-- 
+It is strongly suggested to use `nipkgs-review`:
+- install `nixpkgs-review` for good with `nix-env -f '<nixpkgs>' -iA nixpkgs-review`
+- consult usage documentation: https://github.com/Mic92/nixpkgs-review#usage
+-->
+
+- [ ] Tested sucessful build of final PR `nixpkgs-review pr <pr-number>`.
+      If suceeded, within the resulting `nix-shell`:
+   - [ ] Manually tested execution of all binary files (in `./results/bin/`)
+   - [ ] Mentioned the manual checks and validations at the end of generated `./report.md`
+   - [ ] If ok, posted the results: `nix-shell> nixpkgs-review post-result`
 
-- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
-- Built on platform(s)
+- Platform(s), I built on (reviewers, please complement!):
+<!-- more is better, reviewers might complement -->
    - [ ] NixOS
    - [ ] macOS
-   - [ ] other Linux distributions
-- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
-- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
-- [ ] Tested execution of all binary files (usually in `./result/bin/`)
-- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
+   - [ ] other Linux distributions (Ubuntu, Arch Linux, Alpine, etc.).
+
+- [ ] If available: tested via one or more NixOS test(s) `nixpkgs-review pr -p nixosTests.<test> <package> <pr-number>` (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests)
+<!-- Note, that only few tests are available, if you'd want to write your own have a look at: https://github.com/NixOS/nixpkgs/issues/34987 and furthermore read through https://nixos.org/nixos/manual/index.html#sec-nixos-tests. Looking through the source code should be helpful as well. -->
```

Comment:
Really, the  [installed tests](https://wiki.gnome.org/Initiatives/GnomeGoals/InstalledTests) are only relevant to fraction of GNOME and GNOME-adjacent software, most of which already has a package so I would not waste characters on promoting it here. Saying that as person interested in these tests.